public_dropin_environments/python39_streamlit/Dockerfile

```diff
@@ -11,7 +11,7 @@ RUN apt-get update && apt-get install -y \
 
 COPY ./packages/dr_components-0.1.4-py3-none-any.whl /app/
 
-RUN pip3 install --no-cache-dir 'streamlit==1.31.0' 'streamlit_extras==0.3.6' 'datarobot==3.0.2' 'plotly==5.13.0' 'streamlit-wordcloud==0.1.0' 'kaleido==0.2.1' 'tabulate==0.9.0' 'altair<5'
+RUN pip3 install --no-cache-dir 'streamlit==1.31.0' 'pillow==10.3.0'  'streamlit_extras==0.3.6' 'datarobot==3.0.2' 'plotly==5.13.0' 'streamlit-wordcloud==0.1.0' 'kaleido==0.2.1' 'tabulate==0.9.0' 'altair<5'
```

Comment:
would it make sense to also have `numpy < 2.0.0` in case theres other requirements(.txt) added by user ?